app/config/locale/translations/en.json

```diff
@@ -30,12 +30,6 @@
     "emails.invitation.footer": "If you are not interested, you can ignore this message.",
     "emails.invitation.thanks": "Thanks",
     "emails.invitation.signature": "{{project}} team",
-    "emails.certificate.subject": "Certificate failure for %s",
-    "emails.certificate.hello": "Hello",
-    "emails.certificate.body": "Certificate for your domain '{{domain}}' could not be generated. This is attempt no. {{attempt}}, and the failure was caused by: {{error}}",
-    "emails.certificate.footer": "Your previous certificate will be valid for 30 days since the first failure. We highly recommend investigating this case, otherwise your domain will end up without a valid SSL communication.",
-    "emails.certificate.thanks": "Thanks",
-    "emails.certificate.signature": "{{project}} team",
```

Comment:
Where are we setting the Thanks part? I dont see it in the code below

Reply:
nor in the .tpl file above

Reply:
We are removing it. It used to be set, but our new approach is to not have translation for developer-facing emails.
Ill remove it from all languages, not just EN.